ui/apps/platform/src/Containers/AccessControl/AuthProviders/ConfigurationFormFields.tsx

```diff
@@ -61,14 +62,16 @@ function ConfigurationFormFields({
     type,
     configErrors,
     configTouched,
-    disabled = false,
+    isAuthProviderActive = false,
+    isAuthProviderDeclarative = false,
 }: ConfigurationFormFieldsProps): ReactElement {
     const clientSecretSupported = config.mode !== 'fragment';
     const clientSecretMandatory = config.mode === 'query';
 
     const clientSecretHelperText = getClientSecretHelperText(config, clientSecretSupported);
+    const activeModificationsDisabled = isAuthProviderActive || isAuthProviderDeclarative;
```

Comment:
Although the file lacks consistent name convention for boolean variables, what do you think about `isActiveModificationDisabled` so its many uses look more consistent:

```js
isDisabled={isViewing || isActiveModificationDisabled}
```